setup.sh

```diff
@@ -4,8 +4,13 @@
 . src/kwlib.sh --source-only
 
 # List of dependences per distro
-arch_packages=(qemu bash git tar python-docutils pulseaudio libpulse dunst python-sphinx imagemagick graphviz python-virtualenv texlive-bin librsvg bzip2 lzip lzop zstd xz python-pip)
-debian_packages=(qemu git tar python3-docutils pulseaudio-utils dunst sphinx-doc imagemagick graphviz dvipng python3-venv latexmk librsvg2-bin texlive-xetex python3-sphinx python3-dask-sphinx-theme python3-pip bzip2 lzip xz-utils lzop zstd)
+arch_packages=(qemu bash git tar python-docutils pulseaudio libpulse dunst
+  python-sphinx imagemagick graphviz python-virtualenv texlive-bin librsvg
+  bzip2 lzip lzop zstd xz python-pip)
+debian_packages=(qemu git tar python3-docutils pulseaudio-utils dunst
+  sphinx-doc imagemagick graphviz dvipng python3-venv latexmk librsvg2-bin
+  texlive-xetex python3-sphinx python3-dask-sphinx-theme python3-pip bzip2
+  lzip xz-utils lzop zstd)
```

Comment:
Can we have something like?
```
arch_packages=(
  qemu
  git
  tar
  ...
```

Reply:
Alternatively, how about refactor this so that we add these package's names in a separate file and load them here?
Maybe we can have a file named `archlinux.dependencies` that we load based on the target distro. Another idea is to put this info in the documentation folder as an rst file and load it in the documentation and this script (reuse the same info twice).